source/integrate-with-integration-environment/prove-users-identity.html.md.erb

```diff
@@ -92,9 +142,6 @@ The following are core identity attributes:
 
 <%= warning_text("If the <code>https://vocab.account.gov.uk/v1/coreIdentityJWT</code> property is not present, then GOV.UK One Login was not able to prove your user's identity.") %>
 
-Our support team will give you the public key you must use to validate this JWT. We recommend
-you [use a certified JWT/JWS implementation](https://openid.net/developers/jwt/).
```

Comment:
We probably want to leave in the bit about using a certified JWT/JWS implementation.

We could also link to [DIF](https://identity.foundation/)'s DID implementations: https://github.com/decentralized-identity (though, we might want to check whether we want to do that with _someone_)

Reply:
Agree with regard to leaving the bit about using a certified JWT/JWS implementation

Reply:
Restored